Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -111,6 +111,39 @@ internal static void SeparateOculusFromMRTK()
                 "OK");
         }
 
+        /// <summary>
+        /// Initialize the Oculus Project Config with the appropriate settings to enable handtracking and keyboard support.
+        /// </summary>
+        [MenuItem("Mixed Reality Toolkit/Utilities/Oculus/Initialize Oculus Project Config")]
+        internal static void InitializeOculusProjectConfig()
+        {
+#if OCULUSINTEGRATION_PRESENT
+            // Updating the oculus project config to allow for handtracking and system keyboard usage
+            string[] defaultOculusProjectConfigGuids = AssetDatabase.FindAssets(Path.GetFileNameWithoutExtension(OculusIntegrationProjectConfig));
```

Comment:
Would this be able to call `OVRProjectConfig.GetProjectConfig()` instead of manually searching? It looks like that method  goes and grabs the asset based on the plug-in's internally known path.

Reply:
Though, it also creates one if none exists. Not sure if that's behavior we want to trigger? Though it seems like the asset is required for this feature.

Reply:
That sounds like the right method to use I'll change it